modules/clearAll.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
- [ ] Kindly remove the eslint disable line, it is always better to fix all issues in the code.